src/process.rs

```diff
@@ -51,19 +63,13 @@ pub struct LaunchOptions {
 
 impl LaunchOptions {
     pub fn default_executable() -> Option<std::path::PathBuf> {
-        // TODO BSDs/Unixes are the same?
-        #[cfg(target_os = "linux")]
-        {
-            // TODO Look at $BROWSER and if it points to a chrome binary
-            // $BROWSER may also provide default arguments, which we may
-            // or may not override later on.
+        // TODO Look at $BROWSER and if it points to a chrome binary
+        // $BROWSER may also provide default arguments, which we may
+        // or may not override later on.
 
-            // TODO More paths!?
-            let default_paths = &["/usr/bin/google-chrome-stable"][..];
-            for path in default_paths {
-                if std::path::Path::new(path).exists() {
-                    return Some(path.into());
-                }
+        for app in &["google-chrome-stable", "chromium"] {
```

Comment:
```suggestion
        for app in &["google-chrome-stable", "google-chrome", "chromium"] {
```

On my Linux box it's `google-chrome`.